manifest.yml

```diff
@@ -0,0 +1,7 @@
+---
```

Comment:
I don't see this file installed in README.md or in RPM spec, so presumably this file isn't needed?

Reply:
That's a great catch.  However, in this scenario, the bc_access_menu repository doesn't need to be copied to /var/www/ood/apps/sys at all.  Only the menu item "apps" need to be copied over.  This is a list of directories in my personal ood instance in /var/www/ood/apps/sys.  The ACCESS menu items show up properly.  You'll notice that bc_access_menu is not "deployed" to /var/ww/ood/apps/sys.

activejobs
bc_access_menu_allocations
bc_access_menu_metrics
bc_access_menu_operations
bc_access_menu_support
bc_desktop
bc_osc_codeserver
dashboard
file-editor
files
myjobs
shell


Here's a screenshot of the menu live.

![image](https://user-images.githubusercontent.com/105065813/185166733-eb72fe41-fbd7-42b1-8577-620086530dcb.png)

Reply:
i'm going to remove that manifest file.